docs/reference/cli/openapi/openapi-to-kcl.md

```diff
@@ -1,60 +1,60 @@
 # OpenAPI to KCL
 
-命令
+To convert from models defined in the OpenAPI spec file to KCL schema, we could run the following script:
 
 ```shell
 kcl-openapi generate model -f ${your_open_api_spec.yaml} -t ${the_kcl_files_output_dir}
 ```
 
-示例：
-
-- 输入文件：test_open_api_spec.yaml:
-
-```yaml
-definitions:
-  v1.TestInt:
-    type: object
-    properties:
-      name:
-        type: string
-        format: int-or-string
-    required:
-      - name
-    x-kcl-type:
-      import:
-        package: v1.test_int
-        alias: test_int
-      type: TestInt
-swagger: "2.0"
-info:
-  title: Kusion
-  version: v0.0.2
-paths: {}
-```
-
-- 命令：
-
-```shell
-kcl-openapi generate model -f test_open_api_spec.yaml -t ~/
-```
-
-- 输出：~/models/v1/test_int.k
-
-```python
-"""
-This is the test_int module in v1 package.
-amytestThis file was generated by the KCL auto-gen tool. DO NOT EDIT.
-Editing this file might prove futile when you re-run the KCL auto-gen generate command.
-"""
-
-
-schema TestInt:
-    """v1 test int
-    """
-
-
-    name: int | str
-    """name
-    """
-
-```
+For example:
+
+- There is the OpenAPI spec file that we need to convert: `test_open_api_spec.yaml`
+
+  ```yaml
+  definitions:
+    v1.TestInt:
+      type: object
+      properties:
+        name:
+          type: string
+          format: int-or-string
+      required:
+        - name
+      x-kcl-type:
+        import:
+          package: v1.test_int
+          alias: test_int
+        type: TestInt
+  swagger: "2.0"
+  info:
+    title: Kusion
+    version: v0.0.2
+  paths: {}
+  ```
+
+- The script will be:
+
+  ```shell
+  kcl-openapi generate model -f test_open_api_spec.yaml -t ~/
+  ```
+
+- Then we can find the generated file here: `~/models/v1/test_int.k`
+
+  ```python
+  """
```

Comment:
This example may be outdated. You can check all cases related to the kclopenapi tool.